app/assets/stylesheets/main.css.scss

```diff
@@ -42,9 +37,8 @@ $compatibility-low-color-bg: rgba(178, 58, 44, 0.2);
 $compatibility-med-color-bg: rgba(251, 173, 47, 0.2);
 $compatibility-high-color-bg: rgba(132, 201, 63, 0.2);
 
-@import 'styles/presets';
-
 @import 'styles/foundation_and_overrides';
+@import 'styles/presets';
```

Comment:
Yeah I did it about two hours ago. (this should probably fix the merge conflict)